src/main/java/core/basesyntax/AbstractFigure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+abstract class AbstractFigure implements Figure,Drawable {
```

Comment:
```suggestion
abstract class AbstractFigure implements AreaCalculator, Drawable {
```

Reply:
not fixed(

Reply:
> not fixed(

I redid it as you said in the previous review